cetmix_tower_yaml/wizards/cx_tower_yaml_import_wiz.py

```diff
@@ -16,16 +16,21 @@ class CxTowerYamlImportWiz(models.TransientModel):
         string="Model", readonly=True, compute="_compute_model_description"
     )
     record_id = fields.Integer(readonly=True, help="Record ID to update")
-    update_existing_record = fields.Boolean(
-        default=True,
-        help="If enabled, existing records will be updated with the new data."
-        " Otherwise, new records will be created.",
+    if_record_exists = fields.Selection(
+        selection=[
+            ("update", "Update existing record"),
+            ("create", "Create a new record"),
+        ],
+        default="update",
+        required=True,
+        help="What to do if record with the same reference already exists",
     )
     secret_list = fields.Char(
         readonly=True,
         help="List of secrets present in the YAML file",
         compute="_compute_secret_list",
     )
+    preview_code = fields.Boolean()
```

Comment:
maybe rename to `show_preview_code`

Reply:
In this case "preview" is a verb, so it has the very same meaning)
P.S.  In your case "show_code_preview" because "preview" is a noun here.